packages/client/src/reducers/vaults.js

```diff
@@ -0,0 +1,31 @@
+export const INITIAL_STATE = {
```

Comment:
Suggest renaming to VAULTS_INITIAL_STATE and VaultsReducer for readability and maintainability.

Reply:
I skipped this because it's unified across the whole app, I don't feel like it's affecting readability and maintainability

Reply:
I'm gonna be nit picking here and insist on renaming
Readability is when a person with no former context of our project jumping in and being able to perform static analysis of the code. Imagine yourself as that person and read where the reducer is being called, for example useNFT.js, instead of just telling you we are using the nftReducer, the code directs you to the import statement   , only then you can figure out what's going on. 
I would say there's always room for improvement and optimization even though a pattern is wildly used.

Reply:
Renamed! We will then need to rename the others too, can you please create a ticket for this?

Reply:
created and assigned to myself https://linear.app/dappercollectives/issue/VSL-204